docs/docs/dynamo-store/access-strategies.md

```diff
@@ -57,11 +58,11 @@ Snapshot and how to transform the current state into a Snapshot
 
 ```ts
 const isOrigin = (ev: Event) => ev.type === "Snapshot"
```

Comment:
```suggestion
const isOrigin = (ev: Event) => ev.type === snapshotEventType
```

Reply:
Why though? In TS  `ev.type` has type `"Snapshot" | "Increment"`

Reply:
coz you used the var/constant in other places?
coz it should be `"Snapshotted"`?
to convey that it just gets renamed in all the places if you want to rev the format
hey wait you can't really do that, unless you upconvert from an older format, and typically you just change the category name for that case
hm,...

Reply:
https://github.com/equinox-project/equinox-js/assets/3721521/765f88ba-4dae-4c8d-a41e-5dc9cbb065b1

This ain't you mama's type system 😉

Reply:
replied and it got lost - there was a use of the constant further down - will do final check now...

Reply:
also "Snapshot" should be "Snapshotted" consistetly